### DISCUSSION TITLE
인원이 있는 방에 들어갈시 기존 인원들을 현위치에 생성하는 법

### DISCUSSION THREAD

**Discussion by SweetPotato7432** [naverz/zepeto-studio-kor#939] [5. Q&A]
인원이 있는 방에 테스트 접속시 먼저 들어가있는 인원이 초기 생성위치에서 생성되어 원래 자리로 뛰어갑니다.
각 플레이어들을 화면에 생성할때 현위치를 받아 스폰 시키는 방법이 있을까요?
혹은 다른 플레이어들을 생성하는 코드가 궁금합니다.

**Comment by YSHFM** [comment]
저는 플레이어 위치를 수시로 서버로 보내고 씬 이동을 체크하면 서버에 있는 위치값을 기준으로 클라이언트에 인스턴스 시키고 있습니다.
